automation/src/test/scala/org/broadinstitute/dsde/test/api/SnapshotAPISpec.scala

```diff
@@ -199,6 +199,44 @@ class SnapshotAPISpec extends AnyFreeSpecLike with Matchers with BeforeAndAfterA
       }
     }
 
+    "should delete BigQuery dataset when a snapshot reference is deleted" taggedAs(Tags.AlphaTest, Tags.ExcludeInFiab) in {
```

Comment:
fyi this isn't the lifecycle - we only delete the companion when the *workspace* is deleted, not when a reference is deleted